models/src/cloe/component/utility/steering_utils.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 Robert Bosch GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * \file cloe/component/utility/steering_utils.cpp
+ * \see  cloe/component/utility/steering_utils.hpp
+ */
+
+#include <cassert>
+#include <cmath>
+
+#include <cloe/component/utility/steering_utils.hpp>
+
+namespace cloe {
+namespace utility {
+
+double calculate_wheel_angle(const Geometry& geometry, WheelId wheel_id, double steering_angle) {
```

Comment:
Is there a source somewhere that is publicly available that details why your implmentation is correct? Wikipedia? Or provide some short explanation why this is valid.

Reply:
i added a link to wikipedia (which explains the basic geometry but not the "dynamic part", but from there it is basically math and knowing that the equation is not valid for higher speeds) and a "steering handbook" (which is not publicly available though), but provides the information that it is valid for low speeds and more details. would this be ok?

Reply:
I suppose that's fine. There's probably literature in some library somewhere, but it would be too much to ask to find a reference.